managedtenants/bundles/addon_bundles.py

```diff
@@ -79,3 +136,74 @@ def get_all_bundles(self):
         Returns an addon's main_bundles and dependency_bundles.
         """
         return self.main_bundles + self.dependency_bundles
+
+    def get_all_imagesets(self, index_image):
+        """
+        Produce all imagesets for a given index_image.
+
+        :param index_image_str: representation of an index_image
+        """
+        res = []
+
+        version = self._get_latest_version()
+        ocm_config = self.config.get("ocm", {})
```

Comment:
```suggestion
        ocm_config = self._get_ocm_config()
```

Not sure, but it feels like that defaulting was supposed to be applied here.

Otherwise I don't see where this private method is used outside of tests.

Reply:
good catch, experimented with various things and forget to switch it back